src/box/index.cc

```diff
@@ -199,25 +199,21 @@ check_index(uint32_t space_id, uint32_t index_id,
 }
 
 int
-box_index_tuple_position(uint32_t space_id, uint32_t index_id,
-			 const char *tuple, const char *tuple_end,
-			 const char **packed_pos, const char **packed_pos_end)
+index_def_tuple_position(struct index_def *def, const char *tuple,
```

Comment:
It looks like it's enough to pass `cmp_def` to this function.

Maybe, rename it to `box_iterator_position_from_tuple`?

Reply:
Done